src/objects.js

```diff
@@ -1,29 +1,39 @@
 const createPerson = (name, age) => {
-  // your code here
+  const person = {
+    name: name,
+    age: age,
+  };
+  return person;
 };
 
 const getName = (object) => {
-  // your code here
+  return object.name;
 };
 
 const getProperty = (property, object) => {
-  // your code here
+  return object[property];
 };
 
 const hasProperty = (property, object) => {
-  // your code here
+  return object.hasOwnProperty(property);
 };
 
 const isOver65 = (person) => {
-  // your code here
+  if (person.age > 65) {
```

Comment:
This works, but it is possible without using an if statement. Double check your `boolean.js`.